RetailSuiteAccounting/Operation/10_ManageReceipts.md

```diff
@@ -0,0 +1,194 @@
+
+[comment]: <> (Work in progress!)
+
+
+## Retrieve a receipt
+
+The [ASSIGN RECEIPT] button in the *BOOKINGS* tab allows to retrieve and assign a receipt that has not been registered yet to an existing booking. Any unregistered receipts uploaded in the system are displayed in the *BOOK RECEIPTS* tab.  
+
+After being uploaded, the receipts are stored in the *New documents* folder in the *Documents* module. Once they are booked, they are moved automatically into the *Registered documents* folder and then placed in the appropriate subfolder according to the following structure:
+
+- 10000-69999 personal account numbers in case of a debtors'/customers' document
+- 70000-79999 personal account numbers in case of a creditors'/suppliers' document
+
+
+#### Prerequisites
+
+- The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
+- A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).
+- A manual booking is created, see [Create a manual booking](./04_CreateManualBooking.md).
+- A receipt is uploaded in PDF format in the *Documents* module.
+
+#### Procedure
+
+*Accounting > Select fiscal year > Book > Tab BOOKINGS*
+
+![Assign receipt](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/AssignReceipt.png "[Assign receipt]")
+
+1. Select the booking to be assigned a receipt from the bookings list in the *BOOKINGS* tab.  
+The existing booking details are displayed in the input line fields.
+
+2. Click the [ASSIGN RECEIPT] button.  
+The *Attach Document* window is displayed.
+
+  ![Attach document](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/AttachDocument.png "[Attach document]")
+
+3. Select the document to be assigned to the booking from the *New documents* folder.
+
+  ![Select document](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/AttachDocument01.png "[Select document]")
+
+4. Click the [SELECT DOCUMENT] button.  
+The selected document is now assigned to the booking. A ![Receipt](../../Assets/Icons/Beleg.png "[Receipt]") icon (=Beleg, EN: receipt) will be then displayed next to the booking in the bookings list.
+
+  ![Receipt assigned](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/ReceiptAssigned.png "[Receipt assigned]")
+
+  > [Info] When clicking the ![Receipt](../../Assets/Icons/Beleg.png "[Receipt]") icon, the corresponding receipt is displayed.
+
+5. If the receipt assignment needs to be undone due to a mistake, click the [CANCEL ASSIGNMENT] button.
+
+  ![Attached document](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/AttachedDocument.png "[Attached document]")
```

Comment:
Kein eigener Schritt - ist ja nicht Bestandteil der Procedure, das Assignment wieder zu entfernen. Entweder eigene Procedure zum Löschen eines Assignments oder Einschließen in vorangehende Info